libraries/lib-track/Track.h

```diff
@@ -306,8 +306,8 @@ class TRACK_API Track /* not final */
     @post result: `NChannels() == result->NChannels()`
     */
    virtual TrackListHolder Duplicate(
-      std::optional<std::pair<double, double>> unstretchInterval =
-         std::nullopt) const;
+      std::optional<std::pair<double, double>> unstretchInterval = std::nullopt,
+      std::function<void(double)> reportProgress = [](double) {}) const;
```

Comment:
As I said elsewhere, I would:
- Use type aliases like `using UnstretchInterval = std::pair<double, double>; using ProgressReporter = std::function<void(double)>;` and simplify other usages
- Default the function to '{}', not an empty lambda.  But then wherever the function is ultimately used in WaveTrack or WaveClip, test first that is not null.